datashader/tests/test_spatial.py

```diff
@@ -149,3 +149,15 @@ def test_validate_parquet_file(df, tmp_path):
     spf = dsp.read_parquet(filename)
 
     assert spf.spatial is None
+
+
+def test_filesystem_protocol(df, tmp_path):
+    # Work around https://bugs.python.org/issue33617
+    tmp_path = str(tmp_path)
+    p = 5
+    # Use an filesystem protocol
+    path = "memory://" + os.path.join(tmp_path, 'spatial_points_1.parquet')
```

Comment:
```suggestion
    path = "memory://" + tmp_path + '/spatial_points_1.parquet')
```
(Suggestion from Martin Durant, as memoryfs at least currently expects posix rather than OS-native paths.)

Reply:
Just `path = "memory://tmp_path/spatial_points_1.parquet'` for simplicity. I hope it works!

Reply:
@jrbourbeau , can you please apply the above suggestion to see if it will fix the Appveyor errors?  It seems like it probably will, but I do not have write access to your fork to try it.

Reply:
> "memory://tmp_path/spatial_points_1.parquet'

Here `tmp_path` is a variable, right? Not the literal string "tmp_path"? So I think the path needs to be constructed using code like in my suggestion, not hardcoded like that.

Reply:
Oh, but in that case I guess tmp_path itself may contain backslashes? If so then this suggestion won't work at all, right?

Reply:
It is indeed a variable, but doesn't need to be, since we are to use a temporary in-memory filesystem anyway. I want to avoid anything happening due to string handling, such as "C:\\" messing up what is seen as the "parent" directory. That's not really a test of the filesystem mechanism, since each backend has different definitions of what valid paths look like.

Reply:
Ok.  As I can't edit this, I'll merge and fix it up on master, but I'll have to back out the changes if they don't work out.